src/audiomixerboard.cpp

```diff
@@ -1117,6 +1123,19 @@ void CAudioMixerBoard::ChangeFaderOrder ( const EChSortType eChSortType )
     // sort the channels according to the first of the pair
     std::stable_sort ( PairList.begin(), PairList.end() );
 
+    // move my fader to first position
+    if ( pSettings->bOwnFaderFirst )
+    {
+        for ( int i = 0; i < MAX_NUM_CHANNELS; i++ )
+        {
+            if ( iMyFader == PairList[i].second )
+            {
+                PairList.move ( i, 0 );
+                break;
+            }
+        }
```

Comment:
Not quite happy with the code here bcs. of the break; but it's valid...

Reply:
Why not happy? It seems a right use of `break;` to me: once the correct entry has been found, there is no reason to continue with the loop. I feel the correct use of `break`, `continue` and early `return` contribute greatly to the readability of code and a reduction in unnecessary nesting and conditions.

Reply:
Yeah, probably it’s preference 😀